crates/stackable-operator/src/commons/s3/helpers.rs

```diff
@@ -103,7 +103,7 @@ impl ResolvedS3Connection {
 
         if let Some(credentials) = &self.credentials {
             let secret_class = &credentials.secret_class;
-            let volume_name = format!("{secret_class}-s3-credentials-{unique_identifier}");
+            let volume_name = format!("{unique_identifier}-{secret_class}-s3-credentials");
```

Comment:
This particular change is breaking, right?

Once ready for review, I think there should be some text in a comment ready for adding to release notes (and the `release-note/action-required` label.

Reply:
> This change should be operator-internal and not effect the user.

Sounds good to me.